src/components/Modals/Receive/Receive.tsx

```diff
@@ -10,7 +10,7 @@ import { ReceiveRouter } from './ReceiveRouter'
 export const entries = [ReceiveRoutes.Info, ReceiveRoutes.Select]
 
 type ReceivePropsType = {
-  asset: Asset
+  asset?: Asset
```

Comment:
Do we need to make this optional?

Reply:
I think this was to keep the useModal hook from complaining. 

```receive.open({})```